governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md

```diff
@@ -0,0 +1,133 @@
+discussion_link: https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2024-09-09T16:00:00
+end_date: 2024-09-12T16:00:00
+---
+# Poll: Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024
+
+The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Ecosystem team. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, September 9 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+**This proposal will only take effect if the concurrent Atlas v2 Upgrade proposal passes.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes. In summary, this vovers the following items (find attached corresponding GitHub Repos, which include README files that explain all technical details):
+
+* New Token contracts and corresponding token converters
+  * [USDS](https://github.com/makerdao/sdai/tree/susds) (Upgraded DAI)
+  * [SKY](https://github.com/makerdao/sky) (Upgraded MKR)
+* The introduction of the [Sky Savings Rate (sUSDS)](https://github.com/makerdao/sdai/tree/snst)
+* [Smart Burn Engine](https://github.com/makerdao/dss-flappers/tree/dev) upgrade and [pool migration](https://github.com/makerdao/univ2-pool-migrator) to the new tokens
+* The introduction of [Sky Token Rewards (STR)](https://github.com/makerdao/endgame-toolkit/blob/master/README.md#farms)
+  * Use USDS to get SKY
+  * Use USDS to get 01 rewardsToken
+* [GSM Delay](https://sky-atlas.powerhouse.io/#A.1.8.2_Governance_Security_Delay_Requirements-9349994e-9918-494c-9e9a-e993082ae055%7C0db30758) Changes
+* Corresponding [Atlas V2](https://sky-atlas.powerhouse.io/) Active Data Updates
+
+To learn more about Launch Season, refer to:
+
+* [MakerDAO Endgame: Launch Season (final)](https://forum.makerdao.com/t/makerdao-endgame-launch-season-final/24740)
+* [Sky has arrived!](https://forum.makerdao.com/t/sky-has-arrived/24959)
+* [Atlas V2](https://forum.makerdao.com/t/atlas-v2-upgrade-poll-request/25010)
+* [Endgame: Announcing the Token and Product Launch forum post](https://forum.makerdao.com/t/endgame-announcing-the-token-and-product-launch/25021/1)
+
+### Launch Executive Proposal
+
+* Set the Schedule Method time for the earliest spell execution: September 17, 12:00 UTC
+
+### Deployment Requirements Before Token and Product Launch Spell
+
+* Tokens:
```

Comment:
This list is inconsistent in a similar fashion.

I don't think the requirements are very clear. What does it mean that "Splitter" is required?

Reply:
See above comment